### ISSUE TITLE
Tooltips don't appear with keyboard navigation

### ISSUE THREAD

**Issue by DesktopBookworm** [microsoft/react-native-windows#15417]
### Problem Description

Tooltips only appear on hover with the mouse, but they don't appear when navigating with keyboard. 

### Steps To Reproduce

Navigate with keyboard -> No tooltips are displayed

### Expected Results

Tooltips should be displayed when focusing an item with keyboard navigation

### CLI version

18.0.0

### Environment

```markdown
info Fetching system and libraries information...
System:
  OS: Windows 11 10.0.22631
  CPU: (12) x64 13th Gen Intel(R) Core(TM) i7-1365U
  Memory: 6.89 GB / 31.64 GB
Binaries:
  Node:
    version: 24.5.0
    path: C:\Program Files\nodejs\node.exe
  Yarn:
    version: 1.22.22
    path: C:\Program Files (x86)\Yarn\bin\yarn.CMD
  npm:
    version: 11.6.2
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
  Windows SDK:
    AllowDevelopmentWithoutDevLicense: Enabled
    AllowAllTrustedApps: Enabled
    Versions:
      - 10.0.19041.0
      - 10.0.22000.0
      - 10.0.22621.0
      - 10.0.26100.0
IDEs:
  Android Studio: Not Found
  Visual Studio:
    - 17.14.36623.8 (Visual Studio Professional 2022)
    - 16.11.36631.11 (Visual Studio Professional 2019)
Languages:
  Java:
    version: 2.formatMsgNoLookups
    path: /cygdrive/c/Program Files (x86)/Amazon Corretto/jdk1.8.0_452/bin/javac
  Ruby:
    version: 3.0.2
    path: C:\tools\ruby30\bin\ruby.exe
npmPackages:
  "@react-native-community/cli":
    installed: 18.0.0
    wanted: 18.0.0
  react:
    installed: 19.0.0
    wanted: 19.0.0
  react-native:
    installed: 0.79.4
    wanted: 0.79.4
  react-native-windows:
    installed: 0.79.4
    wanted: 0.79.4
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found

info React Native v0.82.1 is now available (your project is running on v0.79.4).
info Changelog: https://github.com/facebook/react-native/releases/tag/v0.82.1
info Diff: https://react-native-community.github.io/upgrade-helper/?from=0.79.4&to=0.82.1
info For more info, check out "https://reactnative.dev/docs/upgrading?os=windows".
```

### Target React Native Architecture

None

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

**Comment by chrisglein** [comment]
@protikbiswas100 @vivekkhare31 Got a partner ping on status for this. Looks to be scheduled for 0.82. Bug understood or still needs investigation?

**Comment by vineethkuttan** [comment]
For Paper , tooltip was displayed when navigating with keyboard.

We have to fix this , when a component got focus we should display its tooltip.

<img width="1810" height="699" alt="Image" src="https://github.com/user-attachments/assets/e4a05eef-a818-4ed5-ab71-6215ac9d4a65" />
